pennylane/ops/op_math/controlled.py

```diff
@@ -381,9 +381,10 @@ class Controlled(SymbolicOp):
     **Example**
 
     >>> base = qml.RX(1.234, 1)
-    >>> Controlled(base, (0, 2, 3), control_values=[True, False, True])
+    >>> qml.ops.Controlled(base, (0, 2, 3), control_values=[True, False, True])
```

Comment:
Can you actually include an `import pennylane as qml` as the first line of the examples? Many of our documentation examples are missing this.